src/components/card/DueDateSelector.vue

```diff
@@ -203,6 +207,18 @@ export default defineComponent({
 					label: t('deck', 'Next week – {timeLocale}', { timeLocale: nextWeekTime?.format('ddd LT') }),
 					ariaLabel: t('deck', 'Set due date for next week'),
 				},
+				{
+					key: 'nextMonth',
+					timestamp: this.getTimestamp(nextMonthTime),
+					label: t('deck', 'Next month – {timeLocale}', { timeLocale: nextMonthTime?.format('ddd LT') }),
+					ariaLabel: t('deck', 'Set due date for next month'),
+				},
+				{
+					key: 'nextQuarter',
+					timestamp: this.getTimestamp(nextQuarterTime),
+					label: t('deck', 'Next quarter – {timeLocale}', { timeLocale: nextQuarterTime?.format('ddd LT') }),
```

Comment:
Since the target due date might be far from the current date, I think the format should be adjusted to show the year, month, and day as well.